### ISSUE TITLE
Error Submitting Issue: Expected HTTP_CREATED (201), obtained 403 instead..

### ISSUE THREAD

**Issue by Xyndra** [DemonWav/MinecraftDevIntelliJ#2368]
### Minecraft Development for IntelliJ plugin version

2024.2-1.8.1

### IntelliJ version

Build #IU-242.21829.142

### Operating System

Windows 11

### Description of the bug

Error Submitting Issue: Expected HTTP_CREATED (201), obtained 403 instead..

**Comment by Xyndra** [comment]
this is very important please fix

**Comment by DenWav** [comment]
Yeah it's not operating at the moment, I hope to have it back up soon.

**Comment by Xyndra** [comment]
this problem has existed for two weeks wdym you "hope" to have it back up soon???

**Comment by Earthcomputer** [comment]
This problem has existed for far more than two weeks. We had to disable it because someone spammed the error reporter with Nazi messaging.

**Comment by Xyndra** [comment]
and you can't just make a system where you report it using the github account, for example lead them to a link and automatically put the error text in their clipboard for convenience.

**Comment by Earthcomputer** [comment]
Indeed you could, thanks for volunteering!

We would prefer something a bit more automated ideally though

**Comment by Xyndra** [comment]
yeah but how are you gonna avoid the spam situation then? like you can't just implement your own protection since that's practically impossible

**Comment by Xyndra** [comment]
Hi, I wanted to bump this since it has been months and the problem still exists. Please either fix this or provide an easy alternative

**Comment by ArcticWarmth** [comment]
Bumping this. The error still persists
